bindings/python/Makefile

```diff
@@ -1,10 +1,13 @@
 default: build
 
-
-edlib: ../../edlib
-	cp -R ../../edlib .
+.PHONY:
```

Comment:
Why .PHONY? I thought it does not have to be PHONY since it actually creates `edlib` dir, but on the other hand my konwledge of Makefile is very fundamental.

Reply:
I don't know why I wrote that.